pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -363,6 +363,14 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					InputType:    InputTypeText,
 					PropertyName: "details",
 				},
+				{
+					Label:        "API URL",
+					Description:  "API URL of your Pager Duty account",
```

Comment:
let's use the same description 

![image](https://github.com/grafana/grafana/assets/25988953/3a9711a7-08d4-4000-8b24-dd8d2b53af2b)